airdrop/test/airdrop.test.ts

```diff
@@ -0,0 +1,123 @@
+import { Airdrop, AirdropInstance, DisperseAirdrop10 } from '../artifacts/ts'
+import { DUST_AMOUNT, ONE_ALPH, Project, web3 } from '@alephium/web3'
+import { testPrivateKey } from '@alephium/web3-test'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+
+describe('unit tests', () => {
+  const tokenId = '1a281053ba8601a658368594da034c2e99a0fb951b86498d05e76aedfe666800'
+  let airdrop: AirdropInstance
+  let defaultSigner: PrivateKeyWallet
+
+  beforeAll(async () => {
+    web3.setCurrentNodeProvider('http://127.0.0.1:22973', undefined, fetch)
+    await Project.build()
+    defaultSigner = new PrivateKeyWallet({ privateKey: testPrivateKey })
+    // airdrop = (
+    //   await Airdrop.deploy(defaultSigner, {
+    //     initialFields: {
+    //       selfOwner: defaultSigner.address
+    //     }
+    //   })
+    // ).contractInstance
+  })
+
+  it('disperse10', async () => {
+    // const result = await DisperseAirdrop10.execute(defaultSigner, {
+    //   initialFields: {
+    //     airdrop: airdrop.contractId,
+    //     tokenId,
+    //     amountPerAddress: BigInt(1)
+    //   },
+    //   attoAlphAmount: DUST_AMOUNT * BigInt(11),
+    //   tokens: [
+    //     {
+    //       id: tokenId,
+    //       amount: BigInt(10)
+    //     }
+    //   ],
+    // })
+    //
+    // console.log(result)
+
+    const result = await Airdrop.tests.deposit({
+      initialFields: {
+        selfOwner: defaultSigner.address
+      },
+      testArgs: {
+        tokenId,
+        amount: BigInt(10000)
+      },
+      initialAsset: {
+        alphAmount: ONE_ALPH,
+        tokens: [
+          {
+            id: '1a281053ba8601a658368594da034c2e99a0fb951b86498d05e76aedfe666800',
+            amount: BigInt(10000)
+          }
+        ]
+      },
+      inputAssets: [
+        {
+          address: defaultSigner.address,
+          asset: {
+            alphAmount: DUST_AMOUNT * BigInt(11),
```

Comment:
Here, we need to consider gas fee.
If we add more ALPH here the test should pass (you can try adding `ONE_ALPH` for example, which is more than enough)

We will improve the error message :+1: